docs/dev/reference/config.md

```diff
@@ -12,6 +12,8 @@ Contao's configuration options can be reviewed by running the following command:
 $ vendor/bin/contao-console config:dump-reference contao
 ```
 
+The following is a dump of the configuration reference from Contao **5.1**.
```

Comment:
@fritzmg this is not completely accurate as it also includes configuration keys that exist in 4.13 only (like `legacy_routing` for example)
But I thought as a reference having all configurations from 4.13 and 5.1 would be best for the docs.

Reply:
Oh, so you merged them together? We could also make tabs for Contao 4(.13) and 5(.1). That's easier to maintain.

Reply:
> so you merged them together?

Yes.

> We could also make tabs for Contao 4(.13) and 5(.1).

:+1: I will create a new PR for that.

Reply:
Done #1191